src/ducks/apps/components/InstallSuccess.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { PureComponent } from 'react'
+
+import { ModalContent, ModalHeader } from 'cozy-ui/react/Modal'
+import { translate } from 'cozy-ui/react/I18n'
+import AppIcon from 'cozy-ui/react/AppIcon'
+import { fetchIcon } from 'ducks/apps'
+
+export class InstallSuccess extends PureComponent {
```

Comment:
Shouldn't this file be in `src/ducks/components/intents` ? It's only used here.

Reply:
This component is very generic and could be use to replace the success alert in the main application and there is nothing related to the intents in it